evap/staff/forms.py

```diff
@@ -999,6 +1000,7 @@ def clean_email(self):
             user_with_same_email = user_with_same_email.exclude(pk=self.instance.pk)
 
         if user_with_same_email.exists():
+            self.user_with_same_email = user_with_same_email.get()
```

Comment:
This now does two database queries, one for `exists` and one for `get`. In general, could it be the case that multiple users exist? @janno42

Reply:
No, because `email` is unique on database level.

Reply:
Except that we are matching with `iexact` above. I think we can usually assume that all emails in the database are lowercased, but I wonder if we should handle the other case here (or assert that at most one matches?)

Reply:
I think it's fair if we just take the first email that matches. In the (unlikely) case there are multiple matching emails, you'd simply get another merge suggestion afterwards.

Reply:
Done